pkg/pool-utils/contracts/controllers/ManagedPoolController.sol

```diff
@@ -202,10 +206,10 @@ contract ManagedPoolController is BasePoolController, IControlledManagedPool {
     }
 
     /**
-     * @dev Pass a call to ManagedPool's withdrawCollectedManagementFees through to the underlying pool.
+     * @dev Transfer any BPT management fees from this contract to the recipient
      */
     function withdrawCollectedManagementFees(address recipient) external virtual override onlyManager withBoundPool {
-        IControlledManagedPool(pool).withdrawCollectedManagementFees(recipient);
+        IERC20(pool).transfer(recipient, IERC20(pool).balanceOf(address(this)));
```

Comment:
Maybe that's a product question? I was trying to keep it maximally simple, but we could easily just pass in a bptAmount here. If they put in more than the balance, it would just fail.

That would mean callers would need to get the balance before calling this, though, unless we want to make "0" or "maxint" mean transfer all of it, which would need to be documented.